packages/material-ui-styles/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@material-ui/styles",
-  "version": "4.11.4",
+  "version": "4.12.0",
```

We have only changed the tests

```suggestion
  "version": "4.11.4",
```